hypothesis-python/README.md

```diff
@@ -0,0 +1 @@
+The Hypothesis python readme has moved to [the main readme](../README.md)!
```

Comment:
Not quite sure how we want to deal with this :(. 

In general I'm pretty strongly of the opinion that multiple language implementations in the same repository is a bad idea in general, in part because you end up with people linking to /hypothesis/hypothesis-python/ when we really wanted them to link to /hypothesis/...

Reply:
Personally I'd be fine with splitting out the Rust and Ruby implementations; it made sense back when we (briefly) expected to have a shared Rust backend for both frontends, but at this point they're pretty much dead code :/

Reply:
I think doing this makes a lot of sense, I'll leave it for a followup (will probably require org admin perms etc).